test/PooltogetherGovernor.t.sol

```diff
@@ -411,4 +410,497 @@ contract Propose is ProposalTest {
+    _upgradeToBravoGovernor();
+
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildTokenSendProposal(address(_token), _amount, _receiver);
+
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets,
+      _values,
+      _calldatas,
+      _description,
+      (_amount % 2 == 1 ? AGAINST : ABSTAIN) // Randomize vote type.
+    );
+
+    // It should not be possible to queue the proposal
+    vm.expectRevert("Governor: proposal not successful");
+    governorBravo.queue(_targets, _values, _calldatas, keccak256(bytes(_description)));
+  }
+
+  function testFuzz_NewGovernorCanPassProposalToSendToken(
+    uint256 _amount,
+    address _receiver,
+    uint256 _seed
+  ) public {
+    IERC20 _token = _randomERC20Token(_seed);
+    _assumeReceiver(_receiver);
+    uint256 _timelockTokenBalance = _token.balanceOf(TIMELOCK);
+
+    // bound by the number of tokens the timelock currently controls
+    _amount = bound(_amount, 0, _timelockTokenBalance);
+    uint256 _initialTokenBalance = _token.balanceOf(_receiver);
+
+    _upgradeToBravoGovernor();
+
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _buildTokenSendProposal(address(_token), _amount, _receiver);
+
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets, _values, _calldatas, _description, FOR
+    );
+
+    // Ensure the tokens have been transferred
+    assertEq(_token.balanceOf(_receiver), _initialTokenBalance + _amount);
+    assertEq(_token.balanceOf(TIMELOCK), _timelockTokenBalance - _amount);
+  }
+
+  function testFuzz_NewGovernorCanPassProposalToSendETH(uint256 _amount, address _receiver) public {
+    _assumeReceiver(_receiver);
+    vm.deal(TIMELOCK, _amount);
+    uint256 _timelockETHBalance = TIMELOCK.balance;
+    uint256 _receiverETHBalance = _receiver.balance;
+
+    _upgradeToBravoGovernor();
+
+    // Craft a new proposal to send ETH.
+    address[] memory _targets = new address[](1);
+    uint256[] memory _values = new uint256[](1);
+    _targets[0] = _receiver;
+    _values[0] = _amount;
+
+    _queueAndVoteAndExecuteProposalWithBravoGovernor(
+      _targets,
+      _values,
+      new bytes[](1), // There is no calldata for a plain ETH call.
+      "Transfer some ETH via the new Governor",
+      FOR // Vote/suppport type.
+    );
+
+    // Ensure the ETH was transferred.
+    assertEq(_receiver.balance, _receiverETHBalance + _amount);
+    assertEq(TIMELOCK.balance, _timelockETHBalance - _amount);
+  }
+
+  function testFuzz_NewGovernorCanPassProposalToSendETHWithNoDeal(
+    uint256 _amount,
+    address _receiver
+  ) public {
+    _assumeReceiver(_receiver);
+    // The Timelock currently has approximately 30.4 ETH
+    // https://etherscan.io/address/0x42cd8312D2BCe04277dD5161832460e95b24262E
+    vm.assume(_amount < 30.4 ether);
```

Comment:
This should probably use `bound` rather than assume, or you'll get a lot of churn from the fuzzer

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/13/commits/dadc05e6496462c326ff6442758c6fd7b6dd232b